documentation/modules/proc-downloading-product.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-product-downloads-{context}']
+= Downloading {ProductName}
+
+Download a ZIP file that contains the resources required for installation and examples for configuration.
```

Comment:
When I read this, "...and examples for configuration" seems to modify the sentence, suggesting that the examples are required.

Consider "along with" or a similar phrase -- "along with examples for configuration".